app/graphql/mutations/send_push_notification.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Mutations
+  class SendPushNotification < BaseMutation
+    argument :title, String, required: true
+    argument :body, String, required: false
+    argument :force_create, Boolean, required: false
+
+    type Types::StatusType
+
+    def resolve(**params)
+      if params.include?(:title) && params[:title].present?
+        PushNotification.delay.send_notifications(params)
```

Comment:
Hier kann jeder ungesichert an alle Pushmitteilungen versenden wenn man den GraphQL Endpunkt anspricht? `(raise "Access not permitted" if context[:current_user].XXX?)`

Es gibt keine Error Meldung wenn `title.blank?` ?

Der `:body` darf Leer bleiben?

Reply:
36b0cde

<img width="462" alt="image" src="https://user-images.githubusercontent.com/1942953/201317294-917d8c60-1b28-48e2-b6fe-151438490b5a.png">